<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="serialData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: black;
    color: white;
    border-radius: 10px;
    padding: 10px;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_Button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    background-color: #1a1a1a;
    color: #ffffff;
    border-radius: 4px;
    padding: 8px 16px;
    border: none;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #444444;
}

QPushButton:disabled {
    background-color: #666666;
}
</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusIndicator">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(180, 180, 180, 255));
    border: none;
    border-radius: 10px;
    width: 20px;
    height: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DataBase translation</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: black;
    color: white;
    border-radius: 10px;
    padding: 10px;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SerialPort translation</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusIndicator_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(180, 180, 180, 255));
    border: none;
    border-radius: 10px;
    width: 20px;
    height: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Тип Датчика</string>
   </property>
  </action>
  <action name="menu_3">
   <property name="text">
    <string>null</string>
   </property>
  </action>
  <action name="actionn">
   <property name="text">
    <string>n</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Порт</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
